worker/queue.go

```diff
@@ -0,0 +1,54 @@
+package worker
+
+import (
+	pb_worker "dinowernli.me/faucet/proto/service/worker"
+)
+
+// TODO(dino): Find a suitable library and avoid hand-rolling the queue.
```

Comment:
Well, everything is visible inside the package. Right now, "worker" is the only thing using the queue, so no need to export anything outside of the package.

I was thinking of putting the queue in a "collect" package or something, but then it would be awkward to hard-code the payload as an execution request and I'd have to cast things all over the place.